ngraph/frontend/paddlepaddle/src/frontend.cpp

```diff
@@ -67,8 +61,17 @@ namespace ngraph
                     }
                 }
 
-                return CREATORS_MAP.at(op->type())(
-                    NodeContext(DecoderPDPDProto(op_place), named_inputs));
+                try
+                {
+                    return CREATORS_MAP.at(op->type())(
+                        NodeContext(DecoderPDPDProto(op_place), named_inputs));
+                }
+                catch (...)
+                {
+                    // TODO: define exception types
+                    // In case of partial conversion we need to create generic ngraph op here
+                    return NamedOutputs();
```

Comment:
Please don't forget to extend the story to the correct way of cutting and returning here the correct value/re-throwing.